src/main/java/seedu/address/commons/core/AliasMapping.java

```diff
@@ -0,0 +1,140 @@
+//@@author briyanii-reused
+package seedu.address.commons.core;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import seedu.address.logic.commands.ClearCommand;
+import seedu.address.logic.commands.ExitCommand;
+import seedu.address.logic.commands.HelpCommand;
+import seedu.address.logic.commands.ViewHistoryCommand;
+import seedu.address.logic.commands.alias.AliasCommand;
+import seedu.address.logic.commands.issue.AddIssueCommand;
+import seedu.address.logic.commands.issue.DeleteIssueCommand;
+import seedu.address.logic.commands.issue.EditIssueCommand;
+import seedu.address.logic.commands.issue.FindIssueCommand;
+import seedu.address.logic.commands.issue.ListIssueCommand;
+import seedu.address.logic.commands.resident.AddResidentCommand;
+import seedu.address.logic.commands.resident.DeleteResidentCommand;
+import seedu.address.logic.commands.resident.EditResidentCommand;
+import seedu.address.logic.commands.resident.FindResidentCommand;
+import seedu.address.logic.commands.resident.ListResidentCommand;
+import seedu.address.logic.commands.room.AddRoomCommand;
+import seedu.address.logic.commands.room.DeleteRoomCommand;
+import seedu.address.logic.commands.room.EditRoomCommand;
+import seedu.address.logic.commands.room.FindRoomCommand;
+import seedu.address.logic.commands.room.ListRoomCommand;
+
+/**
+ * Represents the current user's {@code Alias} command mapping.
+ * Guarantees: fields are present and not null, fields values are mutable
+ */
+public class AliasMapping implements Serializable {
+    private Map<String, Alias> mapping;
+
+    public AliasMapping() {
+        this.mapping = new HashMap<>();
+    }
+
+    private AliasMapping(AliasMapping aliasMapping) {
+        requireNonNull(aliasMapping);
+        this.mapping = new HashMap<>(aliasMapping.mapping);
+    }
+
+    /**
+     * Returns an Alias object from alias name.
+     * @param aliasName
+     * @return
```

Comment:
Should these `@return` and `@param aliasName` have descriptions?

There are similar cases in this and other files.